spec/ddtrace/contrib/support/spec_helper.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+RSpec.configure do |config|
+  # Raise error when patching an integration fails.
+  # This can be disabled by unstubbing +CommonMethods#on_patch_error+
+  require 'ddtrace/contrib/patcher'
+  config.before(:each) do
+    allow_any_instance_of(Datadog::Contrib::Patcher::CommonMethods).to(receive(:on_patch_error)) { |_, e| raise e }
```

Comment:
Change this to an expectation instead of raising an error.

Reply:
@delner I tried to move this to an expectation in a few different ways but I was not able to remove the expectation when needed (in `spec/ddtrace/contrib/patcher_spec.rb` for example).

I don't think there's a better way that works globally in RSpec at the moment.